### PR TITLE
create-a-new-release-in-jira 1.0.1

### PR DESCRIPTION
**Pull request by Emalober**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ x] __I will not move an already shared step version's tag to another commit__
- [ x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/create-a-new-release-in-jira/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+title: Create a new release in JIRA
+summary: |
+  Create a new release in JIRA.
+description: |
+  This steps create a Release in your JIRA project, with the start date, publication and description you need.
```
Comment (godrei) [inline steps/create-a-new-release-in-jira/1.0.1/step.yml:5]: grammar issue: `This steps create` -> `This Step creates`

File: steps/create-a-new-release-in-jira/1.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  opts:
+    description: |
+      Jira company domain.
+
+      e.g. "example.atlassian.net"
+    is_expand: true
+    is_required: true
+    summary: Jira company domain
+    title: Jira domain
+- jira_user: null
```
Comment (godrei) [inline steps/create-a-new-release-in-jira/1.0.1/step.yml:51]: isn't this input required?

File: steps/create-a-new-release-in-jira/1.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  opts:
+    description: |
+      Jira user used to authenticate with Jira API
+
+      e.g. "someone@example.pl"
+    is_sensitive: true
+    summary: Jira user used to authenticate with Jira API
+    title: Jira user
+- jira_token: null
```
Comment (godrei) [inline steps/create-a-new-release-in-jira/1.0.1/step.yml:60]: isn't this input required?

File: steps/create-a-new-release-in-jira/1.0.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      24 character string
+    is_sensitive: true
+    summary: Jira user token used to authenticate with Jira API
+    title: Jira user token
+- opts:
+    description: |
+      Version description.
+
+      e.g. "An excellent version"
+    is_expand: true
```
Comment (godrei) [inline steps/create-a-new-release-in-jira/1.0.1/step.yml:74]: NOTE: by default `is_expand` is true, so these lines can be omitted

File: steps/create-a-new-release-in-jira/1.0.1/step.yml [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+published_at: 2021-10-07T10:40:41.960748-03:00
+source:
+  git: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira.git
+  commit: 33aff55ee19bd1e089c9bf90273f3ee58fed5ff9
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
```
Comment (godrei) [inline steps/create-a-new-release-in-jira/1.0.1/step.yml:18]: NOTE: this is the default toolkit value, you can omit the whole `toolkit` property this case.

**Comment by jmleruga-technisys** [comment]
Hi, I want to contribute with this step, How can I do it?

**Comment by shams-ahmed** [comment]
hey @jmleruga-technisys, we're glad you like to contribute.. source code live over here: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira

I would suggest creating a discussion or submitting a PR if it's related to this step

If your idea is different from this step theme, I would suggest creating a new step using this guide and template: https://blog.bitrise.io/post/getting-started-with-creating-a-bitrise-step

**Comment by jmleruga-technisys** [comment]
Thank you @shams-ahmed, i will send the PR to the author of the step!

**Pull request merged**